maxi-blocks/maxi-blocks
# issue 2981 container width

# Description

Container needs width option to display in the sidebar if full-width is disabled.

Fixes yeahcan/maxi-blocks#2981 

# How to test?

Add a container and try enabling/disabling full-width from the sidebar. Width option should show only if full-width is disabled.

# Test checklist



**_ Front/Back Testing _**


**_ Pre-Code Testing _**


# Checklist

-   [] I have commented my code, particularly in hard-to-understand areas
-   [] I have added/updated tests that prove my fix is effective or that my feature works
